v3/wdyr.ts

```diff
@@ -0,0 +1,4 @@
+import whyDidYouRender from "@welldone-software/why-did-you-render";
+import React from "react";
+
+whyDidYouRender(React, { trackAllPureComponents: true });
```

Comment:
A couple of minor suggestions:
- rename the file to `why-did-you-render.ts` -- `wdyr.ts` is harder to find or remember
- Add a link to the GitHub repo for the usage instructions